main.tf

```
@@ -197,10 +191,11 @@ resource "google_pubsub_subscription" "push_subscriptions" {
 resource "google_pubsub_subscription" "pull_subscriptions" {
   for_each = var.create_subscriptions ? { for i in var.pull_subscriptions : i.name => i } : {}
 
-  name    = each.value.name
-  topic   = var.create_topic ? google_pubsub_topic.topic.0.name : var.topic
-  project = var.project_id
-  labels  = var.subscription_labels
+  name                         = each.value.name
+  topic                        = var.create_topic ? google_pubsub_topic.topic.0.name : var.topic
+  project                      = var.project_id
+  labels                       = var.subscription_labels
+  enable_exactly_once_delivery = var.enable_exactly_once_delivery
```

I checked the CI example and the `true` wasn't passed through.  As enable_exactly_once_delivery is being passed in the `pull_subscriptions` map, you will probably need something like this:

```suggestion
  enable_exactly_once_delivery = lookup(
    each.value,
    "enable_exactly_once_delivery",
    false,
  )
```